src/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2243,6 +2244,14 @@ public static void TimeZoneInfo_DisplayNameStartsWithOffset()
             }
         }
 
+        [Fact]
+        public static void EnsureUtcObjectSingleton()
```

Comment:
Won't it need a [SkipOnTargetFramework] to exclude NetFx?

Reply:
good point. I'll add the attribute.